packages/ubuntu_bootstrap/lib/services/gnome_accessibility_service.dart

```diff
@@ -55,6 +55,10 @@ class GnomeAccessibilityService implements AccessibilityService {
     }
   }
 
+  @override
+  Future<bool> isSupported() =>
+      _a11yInterfaceSettings.list().then((_) => true, onError: (_) => false);
```

Comment:
In principle we should probably check all gsettings interfaces, but that should do for now :)